src/main/java/cms/gov/madie/measure/resources/BundleController.java

```diff
@@ -30,18 +31,29 @@ public class BundleController {
   public ResponseEntity<String> getMeasureBundle(
       @PathVariable String measureId,
       Principal principal,
-      @RequestHeader("Authorization") String accessToken) {
+      @RequestHeader("Authorization") String accessToken,
+      @RequestParam(required = false, defaultValue = "calculation", name = "bundleType")
+          String bundleType) {
     Optional<Measure> measureOptional = measureRepository.findById(measureId);
-    log.info(
-        "User [{}] is attempting to create a new measure bundle for [{}]",
-        principal.getName(),
-        measureId);
+
+    if ("calculation".equalsIgnoreCase(bundleType)) {
+      log.info(
+          "User [{}] is attempting to create a new measure bundle for [{}]",
+          principal.getName(),
+          measureId);
+    } else {
+      log.info("User [{}] is attempting to export measure [{}]", principal.getName(), measureId);
+    }
     if (measureOptional.isEmpty()) {
       throw new ResourceNotFoundException("Measure", measureId);
     }
     Measure measure = measureOptional.get();
     ControllerUtil.verifyAuthorization(principal.getName(), measure);
 
-    return ResponseEntity.ok(bundleService.bundleMeasure(measure, accessToken));
+    if ("calculation".equalsIgnoreCase(bundleType)) {
+      return ResponseEntity.ok(bundleService.getMeasureBundleForCalculation(measure, accessToken));
+    } else {
+      return bundleService.getMeasureBundleForExport(measure, accessToken);
```

Comment:
this may not be your code but the method name is misleading. it should be `getMeasureExport` or `exportMeasure`

Reply:
@adongare I guess I got the idea from the card that there are two types of bundle, one is for export (which contains more information), and one is for calculation, hence the method names. But I can change them.